src/data/complex/is_R_or_C.lean

```diff
@@ -896,4 +896,34 @@ end
 
 end linear_maps
 
+section nnreal
+open_locale nnreal
+
+/-- The natural coercion from `ℝ≥0` to `K` as a continuous `ℝ≥0`-linear map. -/
+noncomputable def of_nnreal_clm : ℝ≥0 →L[ℝ≥0] K :=
+{ to_fun := algebra_map ℝ≥0 K,
+  cont := is_R_or_C.continuous_of_real.comp continuous_induced_dom,
+  .. (algebra.of_id ℝ≥0 K).to_linear_map }
```

Comment:
```suggestion
  .. (algebra.linear_map ℝ≥0 K) }
```
Isn't this definition relevant more generally for any algebra with continuous scalar multiplication?

Reply:
Ah, I think it would be appropriate to generalize `algebra_map_clm`

Reply:
#16690

Reply:
Is this PR still needed after #16690? If you want a bundled continuous map, that one should also be defined more generally.

Reply:
I'll see how painful it is to use the coercion from `algebra_map_clm` after it merges, and if it's bad like I think I'll define a bundled continuous map in that generality. I'll also need the last few lemmas about the norms in this PR, but I think those should hold more generally too (for any normed algebra over ℝ satisfying `norm_one_class` #16709).

Reply:
As for this PR, I don't think it is needed.